scripts/portfolio.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var projects = [];
+
+function Project (dataIn) {
+  // constructor function for a Project object
+  this.projectTitle   = dataIn.projectTitle;
```

Comment:
These property names seem redundant..  ex: 
```
var projec1t = new Project (data);
console.log(project1.projectTitle);
```
so perhaps shortening them is in order..